### PR TITLE
Month display on graph x-axis 

### LINKED ISSUE #2052: Month displayed in Japanese on graph x-axis on all language
## 起こっている問題 / The Problem
- Month displayed in Japanese on graph x-axis

## スクリーンショット / Screenshot
image

## 期待する見せ方・挙動 / Expected Behavior
image

## 動作環境・ブラウザ / Environment
-  Windows / Android
- Chrome

### PR DESCRIPTION
**Pull request by dg4101**
## 👏 解決する issue / Resolved Issues
- close #2052 

## ⛏ 変更内容 / Details of Changes
- Automatically change month per language locale

## 📸 スクリーンショット / Screenshots
変更前
image

変更後
image

image

## Note
Previously it was hard-coded in *.vue like `month + '月'` 

By adding below line it will automatically pickup current locale and display month accordingly.
`onth.locale(this.$root.$i18n.locale)`
`return month.format('MMM')`
This way there is no need to have additional translation inside locale.json file

### CODE REVIEW

**Comment by kaizumaki** [comment]
@dg4101 Thanks for your contribution! This site uses day.js instead of moment.js. So, I think this PR doesn't fit a little.

**Comment by dg4101** [comment]
Updated to use toLocaleString instead. Please take a look. 
I've seen this is also discussed in #1966.
Please close/revert as needed which fits better to the site overall

File: components/TimeBarChart.vue [hunk 1]
```diff
@@ -291,8 +291,10 @@ const options: ThisTypedComponentOptionsWithRecordProps<
                     'Nov',
                     'Dec'
                   ]
-                  const month = monthStringArry.indexOf(label.split(' ')[0]) + 1
-                  return month + '月'
+                  const mm = monthStringArry.indexOf(label.split(' ')[0]) + 1
+                  const year = new Date().getFullYear()
+                  const mdate = new Date(year + '-' + mm + '-1')
+                  return mdate.toLocaleString(this.$root.$i18n.locale, { month: 'short' })
```
Comment (kaizumaki) [inline components/TimeBarChart.vue]: Please fix this lint error.

File: components/TimeBarChart.vue [hunk 2]
```diff
@@ -291,8 +291,10 @@ const options: ThisTypedComponentOptionsWithRecordProps<
                     'Nov',
                     'Dec'
                   ]
-                  const month = monthStringArry.indexOf(label.split(' ')[0]) + 1
-                  return month + '月'
+                  const mm = monthStringArry.indexOf(label.split(' ')[0]) + 1
+                  const year = new Date().getFullYear()
+                  const mdate = new Date(year + '-' + mm + '-1')
+                  return mdate.toLocaleString(this.$root.$i18n.locale, {month:'short'})
```
Comment (MaySoMusician) [inline components/TimeBarChart.vue]: `this.$root.$i18n.locale` can be `'ja-basic'`, which doesn't adapt to ISO 639 exceptionally. It means やさしい日本語(Simple Japanese) in this project.
Could you pass `'ja'` instead as the first argument to `toLocaleString` when `this.$root.$i18n.locale` is `'ja-basic'`

**Comment by dg4101** [comment]
Made update as requested. Please take a look

**Pull request merged**